pyproject.toml

```diff
@@ -62,7 +62,7 @@ publishing = [
 doc = [
     "mkdocs >=1.4.2",
     "mkdocs-material >=8.2.7",
-    "mkdocstrings[python] >=0.19.1",
+    "mkdocstrings[python] ==0.27.0",
```

Comment:
@raar1 Do we need to make sure to remember this pinned version and to unpin it if the issue is solved in mkdoc? Perhaps create an issue about it?